oracledb-ballerina/client.bal

```diff
@@ -0,0 +1,213 @@
+// Copyright (c) 2021 WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/crypto;
+import ballerina/jballerina.java;
+import ballerina/sql;
+
+public client class Client {
+    *sql:Client;
+    private boolean clientActive = true;
+
+    # Initialize Oracle Client.
+    #
+    # + user - Name of a user of the database
+    # + password - Password for the user
+    # + database - System Identifier or the Service Name of the database
+    # + host - Hostname of the oracle server to be connected
+    # + port - Port number of the oracle server to be connected
+    # + options - Oracle database specific JDBC options
+    # + connectionPool - The `sql:ConnectionPool` object to be used within the jdbc client. If there is no
+    #                    connectionPool provided, the global connection pool will be used
+    public isolated function init(
+        string user,
+        string password,
+        string host = "localhost",
+        int port = 1521,
+        string database = "ORCL",
```

Comment:
Shall we make input parameter order consistent among all db connectors. other's use like,
```
public isolated function init(string host = "localhost", string? user = (), string? password = (), string? database = (),
        int port = 3306, Options? options = (), sql:ConnectionPool? connectionPool = ()) returns sql:Error? {
```

Reply:
Resolved with 171baaf97be39b48c6663ec8edeb3d05944f98f0